bin/load-mocks

```diff
@@ -748,6 +704,9 @@ def main(num_events=1, extra_events=False, load_transactions=False):
 
         OrganizationAccessRequest.objects.create_or_update(member=dummy_member, team=team)
 
+    if load_transactions:
```

Comment:
Any reason to not make this default to on? Having transactions by default simplifies onboarding and daily usage.

Reply:
From personal experience, loading transactions is slow and prone to crashing clickhouse even with the `slow` flag. So I would hold off on making this the default until that's addressed.

Reply:
Wait really? 336 transactions will crash clickhouse locally? Maybe we can load a small handful by default to make at least something appear for onboarding like Mark mentioned, and only load the rest of them to create a trend as their own option.